packages/evm/test/Settler.test.ts

```diff
@@ -286,6 +288,72 @@ describe('Settler', () => {
         })
       }
 
+      const itRevertsUnlessDestinationChain = (reason: string) => {
+        context('when the intent is a swap', () => {
+          beforeEach('set intent type', async () => {
+            intentParams.op = OpType.Swap
+          })
+
+          context('when the swap is single-chain', () => {
+            const sourceChain = 31337
+            const destinationChain = 31337
+
+            beforeEach('set intent data', async () => {
+              intentParams.data = encodeSwapIntent({ sourceChain, destinationChain, tokensIn: [], tokensOut: [] })
```

Comment:
Isn't possible to ignore these?
```suggestion
              intentParams.data = encodeSwapIntent({ sourceChain, destinationChain })
```